docs/supported-technologies.asciidoc

```diff
@@ -50,6 +50,7 @@ These are the frameworks that we officially support:
 so we can't support Koa directly since we rely on router information for full support.
 We currently support the most popular Koa router called https://github.com/alexmingoia/koa-router[koa-router]
 |<<restify,Restify>> |>=5.2.0 |
+|<<fastify,Fastify>> |>=0.27.0 |
```

Comment:
Do we _want_ to advertise support for pre-1.x versions, given the stated instability of that range?

Reply:
🤷‍♂️ our previous approach have been to just show support for the lowest possible versions where our tests still pass without us having to make any crazy changes. If you like I can change it to say 1.0.0 or 2.0.0 though?

Reply:
I'm good either way. Feel free to land, if you are happy with the support range.